python/servo/testing_commands.py

```diff
@@ -31,6 +31,10 @@
 from update import updatecommandline
 import tidy
 
+import site
+site.addsitedir(os.path.abspath(os.path.join(os.path.dirname(__file__), '../tidy_self_test/')))
```

Comment:
In this particular case, I would recommend just turning `python/tidy_self_test/` into a Python module via `touch python/tidy_self_test/__init__.py`. That *should* allow you to do the `import tidy_self_test` here without any `sys.path` overrides.